code/game/objects/buckling.dm

```diff
@@ -63,9 +63,10 @@
 		M.buckling = null
 		return FALSE
 
-	if(M.pulledby)
+	if(M.pulledby.len)
```

Comment:
```suggestion
	if(M.pulledby)
```

Again, you don't need the len check since lists are true if they have >=1 element in them, or false otherwise.

Also, you want to use `length(var)` in general rather than `var.len`.